scala/private/common.bzl

```diff
@@ -38,7 +42,7 @@ def _collect_jars_when_dependency_analyzer_is_off(dep_targets):
   return struct(
       compile_jars = depset(transitive = compile_jars),
       transitive_runtime_jars = depset(transitive = runtime_jars),
-      jars2labels = {},
+      jars2labels = _empty_jars2labels,
```

Comment:
Inlining this seems better for readability, IMHO.

Reply:
Well, I don't know, but I to me, I felt the opposite: it appeared clearer to me (here is an empty value) vs inlining `JarsToLabels(jars_to_labels = {})` which someone has to read an understand as an empty value.

Next, it also seemed it could be faster.

Reply:
But if you would strongly prefer the change, I will make it. Please let me know.